### PR TITLE
[ChatSDK][InlineImage][FileSharing]Update Inline Image and FileSharing Sample

### PR DESCRIPTION
**Pull request by AmyL219**
## Purpose

* Upgrade the dependency of the sample since the newer version is available
*

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by jimchou-dev — APPROVED** [review]

File: join-chat-to-teams-meeting/Chat Experience with File Sharing/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "license": "ISC",
   "dependencies": {
     "@azure/communication-calling": "^1.17.1",
-    "@azure/communication-chat": "1.4.0-beta.2",
+    "@azure/communication-chat": "1.5.0-beta.1",
```
Comment (jpeng-ms) [inline join-chat-to-teams-meeting/Chat Experience with File Sharing/package.json:15]: lock file to be updated when SDK is released

File: join-chat-to-teams-meeting/Chat Experience with File Sharing/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
     "@azure/communication-common": "^2.3.0",
     "@azure/communication-identity": "^1.2.0",
     "@azure/communication-signaling": "1.0.0-beta.21"
```
Comment (emlynmac) [inline join-chat-to-teams-meeting/Chat Experience with File Sharing/package.json:18]: This should be updated too.

File: join-chat-to-teams-meeting/Chat Experience with Inline Image/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "@azure/communication-calling": "^1.17.1",
-    "@azure/communication-chat": "1.4.0-beta.2",
+    "@azure/communication-chat": "1.4.0",
     "@azure/communication-common": "^2.3.0",
     "@azure/communication-identity": "^1.2.0",
     "@azure/communication-signaling": "1.0.0-beta.21"
```
Comment (emlynmac) [inline join-chat-to-teams-meeting/Chat Experience with Inline Image/package.json:17]: This needs an update.

File: join-chat-to-teams-meeting/Chat Experience with File Sharing/client.js [hunk 4]
```diff
@@ -117,7 +117,7 @@ async function renderReceivedMessage(event) {
 
     // Inject image tag for all image attachments
     var imageAttachmentHtml = event.attachments
-        .filter(attachment => attachment.attachmentType === "teamsImage")
+        .filter(attachment => attachment.attachmentType === "image" && !messages.contains(attachment.id))
         .map(attachment => renderImageAttachments(attachment))
         .join('');
     messagesContainer.innerHTML += imageAttachmentHtml;
```
Comment (emlynmac) [inline join-chat-to-teams-meeting/Chat Experience with File Sharing/client.js:123]: Is there a way we can avoid setting the innerHtml and instead injecting DOM nodes to represent the images?

Comment (jpeng-ms) [reply join-chat-to-teams-meeting/Chat Experience with File Sharing/client.js:123]: we should be able to, but we might need some code changes to all 3 quickstarts. I inherited this from original quickstart from Chat team. Feel free to proposal changes if you think it's blocking

**Pull request closed without merging**